base/docs/helpdb.jl

```diff
@@ -328,7 +328,7 @@ Libdl.dlclose
 doc"""
     dlsym_e(handle, sym)
 
-Look up a symbol from a shared library handle, silently return NULL pointer on lookup failure.
+Look up a symbol from a shared library handle, silently return `NULL` pointer on lookup failure.
```

Comment:
@tkelman This (and the corresponding rst update) is the only one could be backported to 0.4.